index_test.go

```diff
@@ -42,15 +43,30 @@ import (
 	"github.com/blevesearch/bleve/index/upsidedown"
 )
 
+type Fatalfable interface {
```

Comment:
I read Fatalf-able as Fatal-Fable :)

Reply:
Yeah at first I didn't like it, but it grew on me.